dali/kernels/erase/erase_cpu.h

```diff
@@ -30,6 +30,22 @@ namespace kernels {
 
 namespace detail {
 
+template <typename O>
+inline std::enable_if_t<std::is_trivially_copyable<O>::value, void>
+CopyImpl(O *out, const O *in, size_t num) {
+  std::memcpy(out, in, num);
+}
+
+template <typename O>
+inline std::enable_if_t<!std::is_trivially_copyable<O>::value, void>
+CopyImpl(O *out, const O *in, size_t num) {
+  for (size_t i = 0; i < num; ++i) {
```

Comment:
With the usage below, you will go out of bounds as `num` is `count * sizeof(O)`

Reply:
Fixed